src/main/java/org/marketdesignresearch/mechlib/outcomerules/ccg/blockingallocation/BlockingCoalitionDetermination.java

```diff
@@ -32,7 +32,7 @@ public BlockingCoalitionDetermination(BundleValueBids<?> bids, Outcome previousO
             Variable traitor = mip.makeNewBooleanVar(TRAITOR + winningBidder.getId());
             BidderAllocation bidderAllocation = previousOutcome.getAllocation().allocationOf(winningBidder);
             BigDecimal payoff = bidderAllocation.getValue().subtract(previousOutcome.getPayment().paymentOf(winningBidder).getAmount());
-            mip.addObjectiveTerm(payoff.negate().doubleValue(), traitor);
+            mip.addObjectiveTerm(payoff.negate().multiply(getScalingFactor()).doubleValue(), traitor);
```

Comment:
This is a left over of the fix (scaling) of the BidBasedWinnerDetermination for MRVM. Can you check if scaling of the objective is here needed? As the objective of the XORWinnerDetermination is scaled as well I think this is needed here.
However, a WinnerDetermination during CCG Outcome computations should never get scaled (As bids are scaled before).

Reply:
Yes, you're right, this is needed. I added a test that catches this. I noted that this is the ORBlockingCoalitionDetermination - For the XORBlockingCoalitionDetermination this part of the logic doesn't seem to be necessary, thus no scaling necessary there. I cannot explain it in detail, it has been too long, but the test confirms it. 😄